### DISCUSSION TITLE
servo speed

### DISCUSSION THREAD

**Discussion by MuffinHavlop** [MastodonRoboticsTeam/VRC2023#14] [Q&A] [answered]
Can anyone summarize for me how to change a servo speed? I forgor

**Comment by ghost** [comment, marked as answer]
I think servo's speed is not fixed and the rpm (rolls per minutes) depends on the voltage you input.. Tbh, you can try setting up the maximum (200) or (-200) if you want the servo roll in inverted way. The range of servo's speed is from -200 to 200 as noted in the first code.
